src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -575,7 +575,7 @@ bool BaseCouplingScheme::measureConvergence(
   PRECICE_ASSERT(_convergenceMeasures.size() > 0);
   if (not utils::MasterSlave::isSlave()) {
     _convergenceWriter->writeData("Timestep", _timesteps);
-    _convergenceWriter->writeData("Iteration", _iterations);
+    _convergenceWriter->writeData("Iterations", _iterations);
```

Comment:
From the results, I understand that this should be `Iteration` (as before), not `Iterations`. For example:

```
Timestep  Iterations  ResNorm(Displacements0)  ResNorm(Forces0)  
1  1  0.0003191866323491  1.0000000000000000  
1  2  0.0003159904961397  0.0000149565768695  
1  3  0.0000000004065084  0.0014792261490834  
2  1  0.0007445657741119  0.2056384940762610  
2  2  0.0000002404722286  0.0039091692413695  
3  1  0.0005485913384467  0.1860456395794740  
3  2  0.0000000664794586  0.0033499436260277  
```
Looking at the `*-convergence.log` file I usually look if the residual norm decreases with every iteration. `Iterations` is not what is meant here, I guess (even if the variable is named `_iterations`).

Am I misunderstanding something?

Reply:
I think that would be right. I vote for keep `TotalIterations` the same and make it `Iteration` (singular, refering to the result of a specific iteration). I don't think changing the variable `_iterations` is necessary.